assets/r/model_build_ctree.R

```diff
@@ -59,7 +60,40 @@ print(model_ctree)
 summary(model_ctree)
 cat("\n")
 
-# Plot the resulting Conditional Inference Tree
+# Plot the resulting Conditional Inference Tree.
+
 svg("TEMPDIR/model_tree_ctree.svg")
 plot(model_ctree, main = paste("Conditional Inference Tree", target))
 dev.off()
+
+# Prepare probabilities for predictions.
+
+predicted_probs <- predict(model_ctree, 
```

Comment:
In the model template we call this pr.

Reply:
Have not find pr in model_template.R

Reply:
Thanks - I probably need to review how closely model_template.R follows the model template from my Essentials book. We can look to update model_template.R from the common code that is emerging from your implementations across the models.